client/my-sites/marketing/main.jsx

```diff
@@ -123,6 +125,9 @@ export const Sharing = ( {
 		titleHeader = translate( 'Integrations' );
 	}
 
+	const upsellNudgeTitle = translate( 'Remove ads from your site with the No Ads add-on' );
+	const upsellNudgeHref = `/checkout/${ siteSlug }/no-ads`;
```

Comment:
This URL won't work (since it's not the standard checkout URL for this product), but I just posted a pull request to make it work :smile: https://github.com/Automattic/wp-calypso/pull/64091

Reply:
Another option is to use the slug used by checkout currently: `no-adverts%25no-adverts.php` (see https://github.com/Automattic/wp-calypso/pull/51909) but I'm fine with a simpler slug also.